browser/ui/webui/settings/brave_account_handler.h

```diff
@@ -6,24 +6,34 @@
 #ifndef BRAVE_BROWSER_UI_WEBUI_SETTINGS_BRAVE_ACCOUNT_HANDLER_H_
 #define BRAVE_BROWSER_UI_WEBUI_SETTINGS_BRAVE_ACCOUNT_HANDLER_H_
 
-#include "base/values.h"
-#include "chrome/browser/ui/webui/settings/settings_page_ui_handler.h"
+#include <string>
 
-class BraveAccountHandler : public settings::SettingsPageUIHandler {
+#include "brave/components/brave_account/core/mojom/brave_account.mojom.h"
+#include "mojo/public/cpp/bindings/pending_receiver.h"
+#include "mojo/public/cpp/bindings/pending_remote.h"
+#include "mojo/public/cpp/bindings/receiver.h"
+#include "mojo/public/cpp/bindings/remote.h"
+
+namespace brave_account {
+class BraveAccountHandler : public mojom::BraveAccountHandler {
  public:
-  BraveAccountHandler();
-  ~BraveAccountHandler() override;
+  BraveAccountHandler(mojo::PendingReceiver<mojom::BraveAccountHandler> handler,
```

Comment:
I feel like we should probably just get rid of BraveAccountClient unless you reckon you'll need it right away

Reply:
Yeah, I do think we'll need it, but I can add it back.